proto/p4/v1/p4runtime.proto

```diff
@@ -471,6 +476,9 @@ message MulticastGroupEntry {
   // Arbitrary metadata from the controller that is opaque to the target.
   // Added in 1.4.0.
   bytes metadata = 3;
+  // Backup replicas used as a fallback port.
+  // Added in 1.4.2.
```

Comment:
My guess is that the next release version will likely be 1.5.0.  The only reason 1.4.1 was released was due to a mistake in release 1.4.0 that we wanted to quickly fix.